### PR TITLE
Added function to calculate P(uplift>0)

### PR DESCRIPTION
**Pull request by jbao**
this solves #2

### CODE REVIEW

File: expan/core/experiment.py [hunk 1]
```diff
@@ -300,36 +300,36 @@ def delta(self, kpi_subset=None, variant_subset=None,
...
+	        relative (boolean): If relative==True, then the values will be
+	            returned as distances below and above the mean, respectively,
+	            rather than the	absolute values. In	this case, the interval is
+	            mean-ret_val[0] to mean+ret_val[1]. This is more useful in many
+	            situations because it corresponds with the sem() and std()
+	            functions.
+
+	    Returns:
+	        Results object containing the computed deltas.
+	    """
```
Comment (robertmuil) [inline expan/core/experiment.py:332]: all of this whitespace change makes it hard to review a PR. Would be better in the future to make these changes in a separate commit, and merge immediately without a PR for that, I think

File: expan/core/results.py [hunk 2]
```diff
@@ -4,40 +4,49 @@
...
 
 from debugging import Dbg
+from pdb import set_trace
+
+mandatory_index_levels = [
+	'metric',
+	'subgroup_metric',
+	'subgroup',
+	'statistic',
+	'pctile']
```
Comment (robertmuil) [inline expan/core/results.py]: Any reason to duplicate this array?

Comment (jbao) [inline expan/core/results.py]: I had to use this later on (line 334) in an external function, maybe we can make this array always global?

Comment (robertmuil) [inline expan/core/results.py]: but it should already be accessible as Results.mandatory_index_levels: it's not an instance variable, just a static class variable

File: expan/core/experiment.py (hunk 1 above)
Comment (robertmuil) [inline expan/core/experiment.py:332]: any chance you could separate the whitespace from the actual change in this PR? I'm struggling to verify it... @jbao

Comment (jbao) [inline expan/core/experiment.py:332]: hmm, no idea now, need to investigate.

File: expan/core/results.py (hunk 2 above)
Comment (jbao) [inline expan/core/results.py]: ah, good to know, i'll reimplement this.

File: expan/core/statistics.py [hunk 3]
```diff
@@ -445,6 +445,25 @@ def normal_difference(mean1, std1, n1, mean2, std2, n2, percentiles=[2.5, 97.5],
 		return dict([(p, mean + stats.t.ppf(p / 100.0, df=d_free) * st_error)
 					 for p in percentiles])
 
+def estimate_std(x, mu, pctile, n1, n2):
+	"""Estimate the standard deviation from a given percentile, according to
+	the formula:
+		x = mu + t * sigma / sqrt(n)
```
Comment (robertmuil) [inline expan/core/statistics.py:451]: that's cool. where did you get this from?

Comment (jbao) [inline expan/core/statistics.py:451]: well, that's basically the inverse step of calculating the CI.

**Comment by robertmuil** [comment]
I approve the merge. Haven't explored possible side-effects fully, nor verified the test cases, but it looks ok.

**Comment by domheger** [comment]
I propose to add a default argument assume_normal=True to calculate_prob_uplift_over_zero() and prob_uplift_over_zero_single_metric() to make this explicit. 
If this parameter is set to False we can raise a NotImplementedError.
Well, actually we use the t-distribution here (I guess because the population std is not known)...

**Comment by jbao** [comment]
@domheger that sounds like a good idea.

**Pull request merged**